### PR TITLE
render name instead of name_en

### PR DESCRIPTION
**Pull request by stephanebisson**

### CODE REVIEW

File: labels.mss [hunk 1]
```diff
@@ -14,7 +14,7 @@
 // - French: '[name_fr]'
```
Comment (MaxSem) [inline labels.mss]: These comments are probably not needed with this change.

**Comment by MaxSem** [comment]
What's the migration plan for these? As of right now, this style would break with old vectors.

**Comment by stephanebisson** [comment]
The migration is part of the i18n deployment. Draft plan is here: https://phabricator.wikimedia.org/T191655

**Comment by pnorman** [comment]
I'm trying to recall if there's a way to do fallback directly in Mapnik with a TextSymbolizer, which would allow the tiles to be upgraded in a compatible manner.

**Comment by stephanebisson** [comment]
I read through https://github.com/mapnik/mapnik/wiki/TextSymbolizer but didn't see a way to configure fallback fields (Look for 'name', then for 'name_en')

**Pull request merged**